byteman.yaml

```diff
@@ -0,0 +1,63 @@
+package:
+  name: byteman
+  version: 4.0.23
+  epoch: 0
+  description: Simplify Java tracing, monitoring and testing with Byteman
+  copyright:
+    - license: GPL-2.0-or-later AND BSD-3-Clause
+
+# NOTE: JRE runtime dependency required but only jar is needed at this point for using this package as part of hadoop.
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - zip
+      - unzip
+      - maven
+      - openjdk-9
```

Comment:
Is `openjdk-9` a typo?

Reply:
RJ, from upstream reference https://github.com/bytemanproject/byteman/blob/4e8bd4985ed3110ff0a2e226e154a72c0d7b8579/bin/bminstall.sh#L40-L52 I found out that only two versions are supported for building it. I first tried it with openjdk-8 and that fails. Later I switched to using openjdk-9 here. 

I also checked the docs but was not able to find supported openjdk version. (ref: https://downloads.jboss.org/byteman/4.0.23/byteman-programmers-guide.html#building-byteman-from-the-sources)

Reply:
JDK 9 is EOL, let's use 11 as it supports 9+